### PR TITLE
Redesign services page

### PR DESCRIPTION
**Pull request by mashooq**
Do not merge yet. The pull request is only for Copy review at the moment.

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -147,42 +147,50 @@ publications-tweet_feed:
 publications-post:
    author_title: By
    date_title: Posted
+services-header:
+   title: Crafted solutions
+   subtitle: Tailored to you
+services-subheader:
+   title: Always listening, constantly evolving
+   subtitle: We understand that everyone is diffrent and will tailor our approach according to the needs of your business and organisation. We will help you build well-crafted software that is valuable to your needs. We provide, <strong>teams</strong> to create valuable, in-time, and quality software, <strong>expert advise</strong> to fast-track your initiatives, and <strong>training and coaching</strong> to better equip your developers with the right skills and attitudes.
```
Comment (jukecraft) [inline _i18n/en.yml]: We understand that everyone is different and will tailor our approach according to the needs of your business and organisation. We will help you build well-crafted software that is relevant to your needs. We provide <strong>teams</strong> to create valuable, in-time, and quality software, <strong>expert advise</strong> to fast-track your initiatives, and <strong>training and coaching</strong> to better equip your developers with the right skills and attitudes.

File: _i18n/en.yml [hunk 2]
```diff
@@ -147,42 +147,50 @@ publications-tweet_feed:
...
 services-software_creation-embedded_craftsmen:
    title: Embedded Craftsmen
    subtitle: Our craftsmen can enhance your teams to provide expertise in specific technologies. They can also form the core of your teams to help you instil XP practices and Software Craftsmanship values from the outset.
 services-software_creation-self_contained_teams:
    title: Self-contained Teams
-   subtitle: <p>Our craftsmen are equipped to cover all the roles required for a fully functioning Agile teams. We have experts in; UX and UI design, Front End and Back End Development, DevOps and Testing. All our craftsmen maintain an innate understanding of business analysis and work with the product owner to ensure that we constantly deliver value.</p>
-           <p>We will provide fully functional teams to help you get started quick and delivery you project in the most effective manner.</p>
+   subtitle: We will provide fully functional teams to help you deliver your project in the most effective and timely manner. Our craftsmen cover all the roles required for an Agile software developement team. We have experts in; UX and UI design, Mobile, Front End and Back End Development, DevOps, and Testing.
```
Comment (jukecraft) [inline _i18n/en.yml]: We will provide fully functional teams to help you deliver your project in the most effective and timely manner. Our craftsmen cover all the roles required for an agile software development team. We have experts in UX and UI design, Mobile, Front End and Back End Development, DevOps culture and Testing.

File: _i18n/en.yml [hunk 3]
```diff
@@ -147,42 +147,50 @@ publications-tweet_feed:
...
+   subtitle: We will help you tailor a Continuous Delivery Pipeline so that your software is responsive to your needs. We are experts in technologies that are enablers for Continuous Delivery. We appreciate how the development team and business need to be aligned to realise the full potential of this approach.
 services-software_creation-ux_ui_design:
    title: UX & UI Design
-   subtitle: We have expertise in designing visual identity and user experiences and understand how this efforts fits best in an Agile environment. We will help you define the brand identity for your project and create UX and UI designs as part of your software development effort.
+   subtitle: We have expertise in designing visual identity and user experiences. We understand how this effort fits best in an Agile Softeware Deveopment environment. We will help you build the brand identity and create UX and UI designs as part of your software development effort.
```
Comment (jukecraft) [inline _i18n/en.yml]: We have expertise in designing visual identity and user experiences. We understand how this effort fits best in an Agile Software Development environment. We will help you build the brand identity and create UX and UI designs as part of your software development effort.

File: _i18n/en.yml [hunk 4]
```diff
@@ -147,42 +147,50 @@ publications-tweet_feed:
...
-           <p>We will review your approach and help you make the most effective decisions for the future.</p>
+   subtitle: Our Craftsmen have a wealth of expertise and experiences in dealing with large scale systems in finance, media and government. We will review your approach and help you make the most effective decisions for the future.
 services-expert_advice-software_architecture_and_design:
    title: Software Architecture & Design
-   subtitle: <p>We will review your current architecture and help you create a risk-based view of your software solution. We can advise on the best way to evolve your architecture to reduce those risks.</p>
-           <p>If you are looking to evolve your system into microservices or create a microservices based architecture from scratch, we will help you understand the best approach according to your needs.</p>
+   subtitle: We will review and help create an apporach to evolve your architecture towards an agreed vision. We have deep knowledge and experience in cloud technologies and enterprise architectures, including Domain Driven Design, Microservices, Event Sourcing and Serverless.
```
Comment (jukecraft) [inline _i18n/en.yml]: We will review and help create an approach to evolve your architecture towards an agreed vision. We have deep knowledge and experience in cloud technologies and enterprise architectures, including Domain Driven Design, Microservices, Event Sourcing and serverless architectures.

File: _i18n/en.yml [hunk 5]
```diff
@@ -147,42 +147,50 @@ publications-tweet_feed:
...
    title: Continuous Delivery
-   subtitle: We are experts in creating continuous delivery pipelines. We understand that it is not only a technical challenge but also an organisational one. We help your customers evolve their approach towards continuous delivery aligning their business, teams and technology.
+   subtitle: We are experts in creating continuous delivery pipelines. We understand that it is not only a technical challenge but also an organisational one. We help our customers evolve their approach towards continuous delivery - aligning their business, teams and technology.
 services-expert_advice-xp:
-   title: Extreme Programming Practices
-   subtitle: XP practices are core to every Craftsman in Codurance. We love to coach and mentor our customers in adopting XP practices. We advise you on how to adopt the practices to bring about positive change in your teams.
+   title: Extreme Programming (XP) Practices
+   subtitle: XP practices are core to every Craftsman in Codurance. We love to coach and mentor our customers in adopting these. We advise on how to adopt XP practices and Softeware Craftsmanship values to bring about positive change in software development teams.
```
Comment (jukecraft) [inline _i18n/en.yml]: XP practices are core to everyone at Codurance. We love to coach and mentor our customers in adopting them. We advise on how to adopt XP practices and Software Craftsmanship values to bring about positive change in software development teams.

File: services/index.html [hunk 6]
```diff
@@ -3,101 +3,157 @@
...
+    <div class="col-md-4" id="training">
+      <div class="panel panel-red equal-height-column">
+
+      <div class="panel-heading" style="background-image:url(/assets/img/custom/services/service-bg-2.png);">
+        <h3 class="panel-title">{% t services-training_coaching.title %}</h3>
+      </div>
+
+      <div class="panel-body">
+          <h4>Software Development Practices</h4>
+          <p>We offer taining to help develpers adopt practices such as Test Driven Development, Continuous Integration, and Release Automation. We can also customise our trainings according to the needs of your team. <a href="#">See our courses ...</a></p>
```
Comment (jukecraft) [inline services/index.html]: We offer training to help developers adopt practices such as Test Driven Development, Continuous Integration, and Release Automation. We can also customise our trainings according to the needs of your team.

File: services/index.html [hunk 7]
```diff
@@ -3,101 +3,157 @@
...
+
+          <h4>Software Architecture and Cloud</h4>
+          <p>We will review your current soution and provide training on archicture and technologies specific to your project. For example, we can train your team to evolve your system towards a Microservices based approach utilising the most suitable offerings from AWS. </p>
```
Comment (jukecraft) [inline services/index.html]: We will review your current solution and provide training on architecture and technologies specific to your project. For example, we can train your team to evolve your system towards a Microservices based approach utilising the most suitable offerings from AWS.

**Review by jukecraft — CHANGES_REQUESTED** [review]
Found a few possible improvements

File: _i18n/es.yml [hunk 8]
```diff
@@ -152,7 +152,7 @@ services-header:
    subtitle: Adaptadas a ti
 services-subheader:
    title: Siempre escuchando, en constante evolución 
-   subtitle: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent ac purus tortor. Praesent et laoreet lorem. Curabitur aliquet magna velit, vitae tincidunt magna laoreet et. Fusce cursus erat at nunc aliquet cursus. Praesent elit tortor, ornare vitae velit at, hendrerit dictum nunc. Nam cursus convallis enim vitae commodo. Nulla facilisi. Nulla quis felis vel risus dignissim sollicitudin ut ut lacus. Curabitur lacus ex, lacinia sed nunc luctus, dignissim tristique massa. Duis in purus sit amet nunc varius finibus id sed.
+   subtitle: Entendemos que cada uno es diferente, por lo que adaptamos nuestro enfoque a las necesidades de cada negocio y organización. Le ayudaremos a construir software bien hecho que sea relevante a sus necesidades. Proporcionamos <strong>equipos</strong> para crear software de calidad, que aporta valor, dentro de las fechas establecidas; <strong>consejo experto</strong> para acelerar sus iniciativas y <strong>formación y coaching</strong> para proveer a sus desarrolladores con las habilidades y actitud necesarias.
```
Comment (srodrigo) [inline _i18n/es.yml]: Actually, on a second thought, maybe "dentro de los plazos establecidos" sounds a bit better in Spanish as a replacement for "in-time". What do you think?

**Comment by alvarogarcia7** [comment]
@franziskas Seems that your suggestions have been applied. Still want to suggest changes or are you OK with the changes now?

**Comment by jukecraft** [comment]
@alvarogarcia7 all good!

**Review by jukecraft — APPROVED** [review]

**Comment by CarlosMChica** [comment]
I just realised that the columns in the "We can help you with" section are not of the same height both in EN & ES.

I'd say that if we can fit the text in the columns to be of the same height the page will look much nicer.

File: services/index.html [hunk 9]
```diff
@@ -133,23 +133,23 @@ <h3 class="panel-title">{% t services-training_coaching.title %}</h3>
...
+											<span>{% t services-section_2.finance %}</span>
 									</a>
 							</div>
 					</div>
 					<div class="col-md-4">
 							<div class="service">
 									<a class="button-with-bg-image"  href="/clients#retail">
 											<img class="img-responsive" src="/assets/img/custom/services/service-bg-2.png" alt="">
-											<span>RETAIL</span>
+											<span>{% t services-section_2.retail %}</span>
```
Comment (srodrigo) [inline services/index.html:144]: The Retail and Media buttons don't look correct due to the length of the text. Would anyone mind to have a look at the styling?
screen shot 2017-01-06 at 09 24 02

Comment (neomatrix369) [reply services/index.html:144]: I'll take a look at this and try to push something as soon as I'm done

Comment (neomatrix369) [reply services/index.html:144]: Looks like @ananogal is already looking at it.

**Pull request merged**